components/style/stylist.rs

```diff
@@ -763,12 +763,12 @@ impl Stylist {
 
         // For most (but not all) pseudo-elements, we inherit all values from the parent.
         let inherit_all = match *pseudo {
+            PseudoElement::ServoAnonymousTableCell |
+            PseudoElement::ServoAnonymousTableRow |
```

Comment:
Please add a comment on how this infliluences stacking context creation and such?

Reply:
I left some comments here. I think they also answer your earlier question about transformations.